### PR TITLE
Add CI publishing, so we never have to run the publish script again.

### PR DESCRIPTION
**Pull request by FredTheDino**
This is easier to work with, since we now don't need to wait for someone who can run the publish script, to run the publish script. It's gonna be great!

Tested at https://github.com/FredTheDino/site-v2/tree/gh-pages

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+name: Website
```
Comment (sornas) [inline .github/workflows/main.yml:1]:
```suggestion
name: website
```

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+            python-version: '3.8'
+      - name: Python Version
+        run: python --version
+      - name: Install PIP
```
Comment (sornas) [inline .github/workflows/main.yml:18]:
```suggestion
      - name: Install pip
```

**Review by sornas — COMMENTED** [review]
Remove some more stuff in `publish`

**Review by sornas — APPROVED** [review]

**Review by EMattfolk — APPROVED** [review]
Looks good. One less thing to worry about.

**Comment by sornas** [comment]
Can confirm that this works :+1:

**Pull request merged**